pkg/cmd/provisioning/ibmcloud/create_service_id.go

```diff
@@ -54,7 +54,7 @@ func NewCreateServiceIDCmd() *cobra.Command {
 
 	createServiceIDCmd.PersistentFlags().StringVar(&CreateOpts.Name, "name", "", "User-defined name for all created IBM Cloud resources (can be separate from the cluster's infra-id)")
 	createServiceIDCmd.MarkPersistentFlagRequired("name")
-	createServiceIDCmd.PersistentFlags().StringVar(&CreateOpts.CredRequestDir, "credentials-requests-dir", "", "Directory containing files of CredentialsRequests to create IAM Roles for (can be created by running 'oc adm release extract --credentials-requests --cloud=aws' against an OpenShift release image)")
+	createServiceIDCmd.PersistentFlags().StringVar(&CreateOpts.CredRequestDir, "credentials-requests-dir", "", "Directory containing files of CredentialsRequests to create IAM Roles for (can be created by running 'oc adm release extract --credentials-requests --cloud=ibmcloud' against an OpenShift release image)")
```

Comment:
This is definitely an improvement, but just out of curiosity: is this already implemented in the recent builds of the oc CLI?

Reply:
yes, it is already in via - https://github.com/openshift/oc/commit/482829c1362784b6e4ea9a6ba3f6f04ac8235a8e